woocommerce/woocommerce-ios
# Product Preview: Limit previews to wp.com stores

Part of https://github.com/woocommerce/woocommerce-ios/issues/7944

## Description

This PR updates product preview flow to enable it only on WP.com stores, because only they are supported for WebView authentication right now.

## How

- Renamed `isWordPressStore` -> `isWordPressComStore` on `Networking.Site` for clarity. There is no change on `Storage.Site` to prevent unnecessary updates in Core Data model.
- Updated `actionButtons` computed property to display preview option only for for sites with `wpcom_store` flag.

## Testing instructions

1. Switch to WP.com store (atomic with mapped domain will work).
2. Open products tab.
3. Open existing draft product or create new one.
4. Confirm the "Preview" button appears in navbar.
5. Switch to self-hosted store.
6. Open existing draft product or create new one.
7. Confirm the "Preview" button doesn't appear in navbar.

---
